bucket/yamTEST/yamOne.yaml

```diff
@@ -0,0 +1,18 @@
+version: 0.8.1
+homepage: https://www.openfaas.com/
+license: MIT
+architecture:
+  64bit:
+    url: https://github.com/openfaas/faas-cli/releases/download/0.8.1/faas-cli.exe
+    hash: 1c8ee6c9aa6d2ff67b5e588442edd7e6291524905abb4eb968f166fc2e45763e
+bin:
+  - faas-cli.exe
+  - - faas-cli.exe
+    - faas
```

Comment:
Looks funny to me.

Reply:
Why would you do that? It would be messy and pretty not straight forward.
You could always use yamls inline one line syntax.

Reply:
```
    "bin": [
        [ "usr/bin/sh.exe", "findssl.sh", "$dir/usr/bin/findssl.sh" ],
        "usr/bin/scp.exe",
        "usr/bin/sftp.exe",
        "usr/bin/ssh.exe",
        "usr/bin/ssh-add.exe",
        "usr/bin/ssh-agent.exe",
        [ "usr/bin/sh.exe", "ssh-copy-id", "$dir/usr/bin/ssh-copy-id" ],
        "usr/bin/ssh-keygen.exe",
        "usr/bin/ssh-keyscan.exe",
        "usr/bin/sshd.exe"
    ],
```
or
```
    "bin": [
        "python.exe",
        [ "python.exe", "python3" ],
        [ "python.exe", "pip", "-m", "pip" ],
        [ "python.exe", "pip3", "-m", "pip" ]
    ],
```
How would it look like in that syntax?

Reply:
That's why yaml is must have for manifests. It's not depending on syntax and it only depends on maintainers, what they prefer in bucket. 

PS: no commas and quotes are needed.

Reply:
And how would it look like? In one-line syntax?

Reply:
https://en.wikipedia.org/wiki/YAML#Basic_components

Basicly as you wrote:


```yaml
bin:
  - single.exe
  - [exe.exe, name, 'some arguments', path to icon ]
```

Reply:
Would one be able to write:
```
  - [exe.exe, name, [/DABC='123 456', arguments], path to icon ]
```
?

Reply:
http://www.yamllint.com/

It could, but it would be agains schema. Since this property is defined as string, array of strings, array of arrays of strings (with 2, 3 or 4 properties)

Reply:
Wait, what is that icon thing? Are you going to merge it as part of ymls? Currently scoop supports any numbers of arguments (for example, 2):
```
        [ "python.exe", "pip", "-m", "pip" ],
```
Additional field (icon?) should be separated from this PR.

Reply:
https://github.com/lukesampson/scoop/blob/f3ddf0c0f81ee2a11466edf5d9f6e38a0fc2b9d4/lib/shortcuts.ps1#L10-L16